views/options/settings.jade

```diff
@@ -88,10 +88,30 @@ script(type='text/ng-template', id='partials/options.settings.settings.html')
           .panel-heading
             span Registration
           .panel-body
-            p(ng-if='user.auth.facebook.id') Registered with Facebook
+            p(ng-if='user.auth.facebook.id') 
+              =env.t('registeredWithFb')
             div(ng-if='user.auth.local.username')
-              p Username: {{user.auth.local.username}}
-              p Email: {{user.auth.local.email}}
+              p
+              =env.t('username') 
+              |: {{user.auth.local.username}}
+              p
+              small
```

Comment:
Anyone else think all these `small` elements should have a `.muted` class applied to them?